prov/util/src/util_attr.c

```diff
@@ -390,6 +390,9 @@ static int fi_resource_mgmt_level(enum fi_resource_mgmt rm_model)
 	}
 }
 
+/* If a provider supports basic registration mode it should set the FI_MR_BASIC
+ * mode bit in prov_mode. Support for FI_MR_SCALABLE is indicated by not setting
+ * any of OFI_MR_BASIC_MAP bits. */
 int ofi_check_mr_mode(uint32_t api_version, uint32_t prov_mode,
 			     uint32_t user_mode)
 {
```

Comment:
We want to check for a specific version, in this case 1.5, not the latest version of the library.  FI_VERSION is almost always going to be correct.